Documentation/scripts/build-marble-graphics.mjs

```diff
@@ -0,0 +1,23 @@
+import { promisify } from 'util';
+import { basename, extname, dirname, join as joinPath } from 'path';
+import { promises as fs } from 'fs';
+import { parseMarbleDiagramSpecification } from '@swirly/parser';
+import { lightStyles } from '@swirly/theme-default-light';
+import { renderMarbleDiagram } from '@swirly/renderer-node';
+import globWithCallback from 'glob';
+
+const glob = promisify(globWithCallback);
+
+const inputFiles = await glob("src/**/*.{swirly,swirl}");
```

Comment:
let's chose a single extension.

Reply:
Addressed in 931d7e9af8938dfe88ced45edaa834185fcd1aa0

Reply:
Addressed in 931d7e9af8938dfe88ced45edaa834185fcd1aa0